build-logic-settings/cache-plugin/src/main/kotlin/convention-cache.settings.gradle.kts

```
@@ -1,18 +1,35 @@
-// See also duplicated settings in parent project
-@Suppress("UnstableApiUsage")
-val avitoGithubRemoteCacheHost: Provider<String> = settings.providers
-    .environmentVariable("GRADLE_CACHE_NODE_HOST")
-    .forUseAtConfigurationTime()
+import com.avito.booleanProperty
+import com.avito.stringProperty
 
-val avitoGithubRemoteCachePush: String =
-    extra.properties.getOrDefault("avitoGithub.gradle.buildCache.remote.push", "false").toString()
+// Duplicated in settings in parent project
+// see https://github.com/gradle/gradle/issues/18511
```

```suggestion
// Can't use only one configuration for cache in the root project.
// Included builds inside `pluginManagement` block can't reuse it, because they are evaluated earlier than the build cache configuration in the root project.
```

The issue will be closed as "won't fix", it can't be fixed.
It'd be more convenient to have details without reading the issue.